README.md

```diff
@@ -103,8 +106,8 @@ Information on how to install Podman in your environment.
 **[OCI Hooks Support](pkg/hooks/README.md)**
 Information on how Podman configures [OCI Hooks][spec-hooks] to run when launching a container.
 
-**[Podman API](API.md)**
-Documentation on the Podman API using [Varlink](https://www.varlink.org/).
+**[Podman API](http://docs.podman.io/en/latest/_static/api.html)**
+Documentation on the Podman HTTP API. Please note that the API is still in its early stages and not yet stable.
```

Comment:
On the fence, but I'm leaning to keeping the varlink bit but marking it as to be deprecated with the V2 release.  It's still usable and supported atm...

Reply:
oops, nvm, should have read further down